ci/cpu/build.sh

```diff
@@ -6,7 +6,10 @@
 set -e
 
 # Set path and build parallel level
-export PATH=/opt/conda/bin:/usr/local/cuda/bin:$PATH
+# FIXME: PATH variable shouldn't be necessary.
+# This should be removed once we either stop using the `remote-docker-plugin`
+# or the following issue is addressed: https://github.com/gpuopenanalytics/remote-docker-plugin/issues/47
+export PATH=/usr/local/gcc9/bin:/opt/conda/bin:/usr/local/cuda/bin:$PATH
```

Comment:
Yes. The value of the `PATH` variable that I added here is the value that we hardcode into our CI images since we use custom compilers for our packages. However, because of the issue in the plugin mentioned in my comment (which is used in CI), the `PATH` variable gets overwritten in CI. This was causing issues for the `libcudf_example` package, which was picking up the CentOS 7 default version of `gcc`, which is something like `4.x.x` instead of `our custom 9.x.x` version.